src/views/shared/fiat-amount/fiat-amount.view.tsx

```diff
@@ -3,13 +3,14 @@ import React from "react";
 import { CurrencySymbol, formatFiatAmount, isValidCurrencySymbolKey } from "src/utils/currencies";
 
 type FiatAmountProps = {
-  amount?: number;
   currency: string;
+  amount?: number;
+  className?: string;
 };
 
 function FiatAmount({
```

Comment:
Just an error from the past in this function. The signature should be:

```ts
function FiatAmount({
  currency,
  amount,
  className,
}: FiatAmountProps): JSX.Element { //... }
```

Can you update it here please?